x/evm/core/vm/custom_eip.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package vm
+
+import (
+	"fmt"
+	"sort"
+)
+
+// ExtendActivators allows to merge the go ethereum activators map
+// with additional activators.
+func ExtendActivators(eips map[int]func(*JumpTable)) error {
+	// Catch early duplicated eip.
+	keys := make([]int, 0, len(eips))
+	for k := range eips {
+		if ExistsEipActivator(k) {
+			return fmt.Errorf("duplicate activation: %d is already present in %s", k, ActivateableEips())
+		}
+		keys = append(keys, k)
+	}
+
+	// Sorting keys to ensure deterministic execution.
+	sort.Ints(keys)
+
+	for _, k := range keys {
+		activators[k] = eips[k]
+	}
+	return nil
+}
+
+func GetActivatorsEipNumbers() []int {
+	keys := make([]int, 0, len(activators))
+	for k := range activators {
+		keys = append(keys, k)
+	}
```

Comment:
you can use the spead operator

Reply:
activators is a Map, we have to range to get keys